### PR TITLE
Project Clue

### PR DESCRIPTION
**Pull request by Sara-Borgstrom**

### CODE REVIEW

**Comment by Sara-Borgstrom** [comment]
clue-sara.netlify.com

File: code/clue.js [hunk 1]
```diff
@@ -1,25 +1,178 @@
...
+  weight: 2,
+  image: "weapons/throphy.png"
+}
+const pistol = {
+    name: "Pistol",
+    material: "Steel",
+    weight: 6,
+    image: "weapons/pistol.png"
+  }
+  // THE ROOMS
```
Comment (ericaglimsholt) [inline code/clue.js:114]: Nice initiative to add images to the rooms!

File: code/clue.js [hunk 2]
```diff
@@ -29,36 +182,120 @@ const suspects = [
...
+const pickRoom = () => {
+  mystery.room = randomSelector(rooms);
+
+  const theRoom = document.getElementById("rooms");
+  const theRoomName = document.getElementById("roomName");
+  const theRoomImage = document.getElementById("roomImage");
+
+  theRoom.style.background = "#a98";
+
+  theRoomName.innerHTML = `${mystery.room.name}`;
```
Comment (ericaglimsholt) [inline code/clue.js:286]: Hm, I don't think that you need to it this way, just typing `mystery.room.name` should be enough

File: code/index.html [hunk 3]
```diff
@@ -1,35 +1,52 @@
...
+    <div class="final">
+      <h2 id="mystery"></h2>
+      <button onclick="revealMystery()">REVEAL THE CRIME</button>
+    </div>
+  </section>
+
+  <script src="clue.js"></script>
+
+  <footer>
+    <p class="cread">Illustrations from Nuon Project by: David Carrero, Phạm Thanh Lộc, Andrejs Kirma, Adib Muhammad, Seona Kim,
```
Comment (ericaglimsholt) [inline code/index.html:44]: Love seeing this! 🙌🏻

**Review by ericaglimsholt — APPROVED** [review]
I think you did a good job! 👏🏻I can see that you have started with some responsive behaviour. Would of course be nice to see it more responsive than what it is right now, but I know that you maybe don't always have the time!

**Pull request closed without merging**